src/main/kotlin/org/elm/lang/core/psi/elements/ElmValueDeclaration.kt

```diff
@@ -48,6 +48,9 @@ class ElmValueDeclaration : ElmStubbedElement<ElmValueDeclarationStub>, ElmDocTa
     val pattern: ElmPattern?
         get() = findChildByClass(ElmPattern::class.java)
 
+    /** The element on the left-hand side of the `=` */
+    val assignee: ElmValueAssigneeTag?
```

Comment:
This tag and property didn't end up being critical. I can remove them if you'd rather keep things simple.